packages/eslint-plugin-template/tests/rules/attributes-order/cases.ts

```diff
@@ -338,4 +338,84 @@ export const invalid = [
                    
     `,
   }),
+  convertAnnotatedSourceToFailureCase({
+    messageId,
+    description: 'should work with implicit structural directive',
+    annotatedSource: `
+       <div test-attr *colDef="let element"></div>
+            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    `,
+    data: {
+      actual: '`test-attr`, `*colDef`',
+      expected: '`*colDef`, `test-attr`',
+    },
+    annotatedOutput: `
+       <div *colDef="let element" test-attr></div>
+            
+    `,
+  }),
+  // convertAnnotatedSourceToFailureCase({
```

Comment:
@JamesHenry can you help me with this, I cannot make this work.

Reply:
Hey @skrtheboss, try the following:
```ts
  convertAnnotatedSourceToFailureCase({
    messageId,
    description: 'should work with aliased structural directive',
    annotatedSource: `
       <div test-attr *ngIf="sth.property as property"></div>
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    `,
    data: {
      actual: '`test-attr`, `*ngIf`',
      expected: '`*ngIf`, `test-attr`',
    },
    annotatedOutput: `
       <div *ngIf="sth.property as property" test-attr></div>
            
    `,
  }),
  ```
  The key here is the amount of whitespace in the newline of the `annotatedOutput`; enough to reach where the `~` annotation began in the reporting.